### PR TITLE
[Mailer] Add mailomat mailer

### PR DESCRIPTION
**Pull request by plandolt**
Relates to https://github.com/symfony/symfony/pull/57456

### CODE REVIEW

File: mailer.rst [hunk 1]
```diff
@@ -115,9 +116,9 @@ Service               Install with                                    Webhook su
 `SendGrid`_           ``composer require symfony/sendgrid-mailer``    yes
 ===================== =============================================== ===============
 
-.. versionadded:: 7.1
+.. versionadded:: 7.2
 
-    The Azure and Resend integrations were introduced in Symfony 7.1.
+    The Mailomat integration was introduced in Symfony 7.2.
```
Comment (OskarStark) [inline mailer.rst]: We need to keep 7.1 versionadded, please addd one for 7.2 below

File: webhook.rst [hunk 2]
```diff
@@ -27,14 +27,15 @@ Brevo          ``mailer.webhook.request_parser.brevo``
...
 Mailgun        ``mailer.webhook.request_parser.mailgun``
 Mailjet        ``mailer.webhook.request_parser.mailjet``
+Mailomat       ``mailer.webhook.request_parser.mailomat``
 Postmark       ``mailer.webhook.request_parser.postmark``
 Resend         ``mailer.webhook.request_parser.resend``
 Sendgrid       ``mailer.webhook.request_parser.sendgrid``
 ============== ============================================
 
-.. versionadded:: 7.1
+.. versionadded:: 7.2
```
Comment (OskarStark) [inline webhook.rst:40]: same as above

File: mailer.rst (hunk 1 above)
Comment (plandolt) [reply mailer.rst]: changed accordingly.

File: webhook.rst (hunk 2 above)
Comment (plandolt) [reply webhook.rst:40]: changed accordingly.

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Patrick.

**Pull request merged**